app/controllers/items/new.js

```diff
@@ -593,6 +615,15 @@ export default GoodcityController.extend(
         });
       },
 
+      setPrinterValue(value) {
+        this.set("selectedPrinterDisplay", {
+          id: value.id,
+          tag: this.get("store")
```

Comment:
```suggestion
          tag: this.get("selectedPrinterDisplay").tag
```
No need of peekRecord. You should use the property that you defined above viz `selectedPrinterDisplay`.